src/main/java/com/powsybl/openloadflow/NetworkCache.java

```diff
@@ -175,6 +172,32 @@ private boolean onSwitchUpdate(String switchId, boolean open) {
             return found;
         }
 
+        private static void updateSwitch(boolean open, LfNetwork lfNetwork, LfBranch lfBranch) {
+            var connectivity = lfNetwork.getConnectivity();
+            connectivity.startTemporaryChanges();
+            try {
```

Comment:
I think we must do something to mutualise the connectivity management. We have almost the same treatment in LfAction, and it is not the first time we forgot to update connectivity. Any idea?